xtask/src/main.rs

```diff
@@ -584,6 +585,9 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 std::env::set_var("XOUS_SVD_FILE", path.canonicalize().unwrap());
                 println!("XOUS_SVD_FILE: {}", path.canonicalize().unwrap().display());
             }
+            let generated_path = std::path::Path::new("utralib/src/generated.rs");
```

Comment:
I'm switching between two UTRA configurations (one with profiling and one without) and sometimes the file isn't being regenerated properly. This clears the file without deleting it (NB the file must exist for the generation routine to work, so just deleting it would cause everything to fail), which seems to fix the issue.